driver-core/src/test/unit/com/mongodb/internal/connection/LoadBalancedClusterTest.java

```diff
@@ -317,6 +321,25 @@ public void shouldTimeoutSelectServerAsynchronouslyWhenThereIsSRVLookupException
                 exception.getMessage());
     }
 
+    @Test
```

Comment:
Could we also have a `shouldCloseServerWhenClosing` test?

Reply:
Sure, added in fcc97d288137284b86c588955daa3b876c949f5c.